modules/nw-ingress-edge-route-default-certificate.adoc

```diff
@@ -0,0 +1,51 @@
+// This is included in the following assemblies:
+//
+// networking/routes/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="creating-edge-route-with-default-certificate_{context}"]
+= Creating a route using the default certificate through an ingress object
+
+If you create an ingress object without specifying any TLS configuration, {product-title} generates an insecure route.  To create an ingress object that generates a secure, edge-terminated route using the default ingress certificate, you can specify an empty TLS configuration as follows.
+
+.Prerequisites
+* You must have a service that you want to expose.
+
+.Procedure
+
+. Define an ingress object in the {product-title} console or by entering the `oc create` command:
++
+[source,yaml]
```

Comment:
Thanks!  I've added a label and split the procedure into two steps: create the yaml file and create the object by running `oc create` with that yaml file.  Is that what you had in mind?  I also added a third step for verification.

Reply:
> Is that what you had in mind?

yep! It looks great, thank you

> I also added a third step for verification

Thank you! These are always good to have. They are usually separated out from the procedure proper, e.g. like it is [here](https://github.com/openshift/openshift-docs/pull/49786/files#diff-40cf60c12328ca3f86d22318a76ad770e0621924d2fdf2b2adba2e16e14a859dR214). A lot of verification sections are single step, so it would also use an unordered list.